### PR TITLE
test(e2e) MQ-2644 invalid volume sizes

### PR DESCRIPTION
**Pull request by milanhajek**
I have issue with test 2 and 3. I can't find out what is wrong. Clean up job can't delete PVC

### CODE REVIEW

**Comment by chriswldenyer** [comment]
> I have issue with test 2 and 3. I can't find out what is wrong. Clean up job can't delete PVC

Hi Milan, I think the problem is that the fio pod using the PVC still exists (even though it has completed). I think in your test you need to delete the fio pod (which removes the finalizer on the PVC) before the PVC can be deleted. HTH

**Review by blaisedias — COMMENTED** [review]
I tried checking out the branch running your test.
In general (very few exception) tests are written such that on successful completion resources instantiated during the test run are destroyed.
This means that the test pods  volumes created for testing must be destroyed.

File: src/MQ-2644-invalid_volume_sizes/invalid_volume_sizes_test.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	BeforeEach(func() {
+		// Check ready to run
+		testNames = nil
+		err := k8stest.BeforeEachCheck()
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	AfterEach(func() {
+		// Check resource leakage.
+		cleanUp()
```
Comment (rohan2794) [inline src/MQ-2644-invalid_volume_sizes/invalid_volume_sizes_test.go:38]: All resource created in every It should be cleared at the end of particular case.

File: src/MQ-2644-invalid_volume_sizes/config.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	storageV1 "k8s.io/api/storage/v1"
+	"mayastor-e2e/common"
+	"mayastor-e2e/common/e2e_config"
+)
+
+const (
+	defTimeoutSecs = 3600
+)
+
+var testNames []string
```
Comment (rohan2794) [inline src/MQ-2644-invalid_volume_sizes/config.go:14]: no need of  testNames this if you only delete resources at the end of every It clause

**Review by chriswldenyer — APPROVED** [review]
LGTM

**Review by rohan2794 — APPROVED** [review]
LGTM

**Pull request merged**